java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -138,4 +141,33 @@ public CompletableFuture<String[]> time() {
     public CompletableFuture<Long> lastsave() {
         return commandManager.submitNewCommand(LastSave, new String[0], this::handleLongResponse);
     }
+
+    @Override
+    public CompletableFuture<String> lolwut() {
+        return commandManager.submitNewCommand(LOLWUT, new String[0], this::handleStringResponse);
+    }
+
+    @Override
+    public CompletableFuture<String> lolwut(int[] parameters) {
```

Comment:
Should this be using `@NonNull`?
```suggestion
    public CompletableFuture<String> lolwut(int @NonNull [] parameters) {
```